felix/fv/flow_logs_goldmane_staged_test.go

```diff
@@ -325,8 +325,8 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ goldmane flow log with stag
 
 		if !bpfEnabled {
 			// Wait for felix to see and program some expected nflog entries, and for the cluster IP to appear.
-			Eventually(getRuleFunc(tc.Felixes[0], "APE0|default.ep1-1-allow-all"), "10s", "1s").ShouldNot(HaveOccurred())
-			Eventually(getRuleFunc(tc.Felixes[1], "APE0|default.ep1-1-allow-all"), "10s", "1s").Should(HaveOccurred())
+			Eventually(getRuleFunc(tc.Felixes[0], "APE0|GlobalNetworkPolicy/default.ep1-1-allow-all"), "10s", "1s").ShouldNot(HaveOccurred())
```

Comment:
First time I've clocked what the rules look like; is it OK to bloat the prefix with the full type name? We've been pretty terse with the other prefixes, maybe that has a slight perf impact (or maybe we're just trying to leave as much room as possible for the policy name)?

Could compress it to a single char `G=GNP` `N=NP` `K=K8s NP`, ...?